nativelib/src/main/resources/gc/immix/headers/ObjectHeader.h

```diff
@@ -0,0 +1,113 @@
+#ifndef IMMIX_OBJECTHEADER_H
+#define IMMIX_OBJECTHEADER_H
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include "../GCTypes.h"
+#include "../Constants.h"
+#include "../Log.h"
+
+typedef enum {
+    object_forward = 0x0,
```

Comment:
This doesn't seem to be used at the moment, I assume that's the state you set for objects that are being evacuated to a different location?